test/e2e/operator.go

```diff
@@ -402,13 +412,15 @@ var _ = Describe("ARO Operator - MUO Deployment", func() {
 })
 
 var _ = Describe("ARO Operator - MHC Deployment", func() {
-	Specify("MHC should be enabled and managed by default", func() {
+	It("must be deployed and managed by default", func() {
```

Comment:
I think this test (and probably some other "by default" tests) are a bit odd. I think it should be an unit test, not an e2e.

I decided not to touch it for now: just add splitting into steps.

Reply:
I agree.  Unless we're testing the actual reconciliation of a controller (creating, deleting, updating resources) by changing the flags, we shouldn't have these.  

If you want to remove them in another PR, be my guest :rocket: